pkg/controller/networkpolicy/clusternetworkpolicy.go

```diff
@@ -254,15 +258,154 @@ func (n *NetworkPolicyController) deleteNamespace(old interface{}) {
 	}
 }
 
+// filterACNPsByServiceAccount gets all ClusterNetworkPolicies that will need to be
+// re-processed based on if they are affected by an added/updated/deleted
+// ServiceAccount.
+func (n *NetworkPolicyController) filterACNPsByServiceAccount(sa *v1.ServiceAccount) (acnps []*crdv1alpha1.ClusterNetworkPolicy) {
+	acnpObjs, err := n.cnpInformer.Informer().GetIndexer().ByIndex(HasServiceAccountsPeerIndex, strconv.FormatBool(true))
```

Comment:
The other place following Antonin's comment where "true" is enough (or maybe let's define a const?)

Reply:
Done.

Reply:
I didn't define it as a const. Because I think we could also change the impl of `PerNamespaceRuleIndex` to follow how we impl `HasServiceAccountsPeerIndex` and define the const in that PR. What do you think?

Reply:
Yeah this is a nit that doesn't need to be addressed